rotkehlchen/api/v1/resources.py

```diff
@@ -815,6 +816,11 @@ def delete(
     ) -> Response:
         return self.rest_api.delete_rpc_node(identifier=identifier, blockchain=blockchain)
 
+    @require_loggedin_user()
+    @use_kwargs(ConnectToRPCNodes(), location='json_and_query_and_view_args')
```

Comment:
can you do it like the rest and do a class variable called `post_schema = ConnectToRPCNodes()` and use it here?